climada/util/api_client.py

```diff
@@ -844,6 +844,44 @@ def get_litpop_default(self, country=None, dump_dir=SYSTEM_DIR):
             raise ValueError("country must be string or list of strings")
         return self.get_exposures(exposures_type='litpop', dump_dir=dump_dir, properties=properties)
 
+    def get_centroids(self, res_arcsec_land=150, res_arcsec_ocean=1800, extent=None, reg_id=None, country_iso3alpha=None,
```

Comment:
I am not sure whether the attribute `reg_id` is very sensible here. This is a number that cannot really be known by a user of the API right? Anyway, the `reg_id` is the same as the `country_iso3alpha` correct? I would thus keep only one of the two, and only the one that is really understandable by a user.

Reply:
okay, it was just to follow the centroids.select() arguments. but probably country_iso3alpha is sufficient